src/server/JasmineGraphServer.cpp

```diff
@@ -1510,6 +1533,99 @@ void JasmineGraphServer::removeGraph(vector<pair<string, string>> hostHasPartiti
     }
 }
 
+int JasmineGraphServer::removeFragmentThroughService(string host, int port, string graphID, string masterIP) {
+    /**
```

Comment:
Change the function documentation following this [URL](https://developer.lsst.io/v/DM-5063/docs/cpp_docs.html#function-method-definitions).

Reply:
Followed the guidelines and moved the multi-line comment above the function signature. Used `@param` to before describing each input parameter. Removed "INPUT" and "OUTPUT" parts of the function documentation comment

Reply:
Ok